lib/protein/get_const.rb

```diff
@@ -2,17 +2,18 @@ module Protein
 class GetConst
   class << self
     def call(input)
-      if input.is_a?(String)
+      case input
+      when nil
+        raise DefinitionError, "unset required option"
+      when String
         Object.const_get(input)
-      elsif input != nil
-        input
       else
-        raise DefinitionError, "unset required option"
+        input
       end
     end
 
     def map(array)
-      array.map { |input| call(input) }
+      array.map(&method(:call))
```

Comment:
Why not `&:call`?

Reply:
Because we need to call `self.call`, and not `array.each.call`.